addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/keypad.xml

```diff
@@ -51,6 +51,14 @@
 				<label>Keypad AC LED</label>
 				<description>Keypad AC LED (0=Off, 1=On, 2=Flashing)</description>
 			</channel>
+            <channel id="keypad_lcd_update" typeId="message">
```

Comment:
Yes, for Java code we use spaces, but for XML we decided to go for tabs, since these files are packaged up in the binary and tabs will safe a few bytes in size here.

You find the XML setting (see "Indent using tabs") here:

<img width="689" alt="screen shot 2016-11-12 at 21 18 55" src="https://cloud.githubusercontent.com/assets/3244965/20240647/dd3430b4-a91d-11e6-8c2d-f02d84d7d5a9.png">